cypress/e2e/tests/migration/task-manager/task_manager_drawer.test.ts

```diff
@@ -0,0 +1,101 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteApplicationTableRows,
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+} from "../../../../utils/utils";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+import { taskNotificationBadge } from "../../../views/common.view";
+
+const analyses: Analysis[] = [];
+const NUMBER_OF_APPS = 2;
+
+function validateTasks(taskType: string, expectedTasks: string[]) {
+    cy.get("h2.pf-v5-c-notification-drawer__list-item-header-title").each((item) => {
+        cy.wrap(item)
+            .invoke("text")
+            .then((text) => {
+                if (item.text().includes(taskType)) {
+                    // Extract Task ID from the drawer item title; Task ID is present at the start of the string
+                    const match = text.match(/^\d+/);
+                    if (match) {
+                        const taskID = parseInt(match[0], 10);
+                        expect(Number.isInteger(taskID), "Task ID should be an integer").to.eq(
+                            true
+                        );
+                    }
+
+                    // Assert that drawer item title contains task type, app name and task priority
+                    expect(
+                        text.replace(/^\d+\s/, ""),
+                        "Task should be in expected list"
+                    ).to.be.oneOf(expectedTasks);
+                }
+            });
+    });
+}
+
+describe(["@tier2"], "Task drawer validation", () => {
+    before("Login", function () {
+        login();
+        cy.visit("/");
+        deleteApplicationTableRows();
+        cy.fixture("application").then((appData) => {
+            cy.fixture("analysis").then((analysisData) => {
+                for (let i = 0; i < NUMBER_OF_APPS; i++) {
+                    analyses.push(
+                        new Analysis(
+                            getRandomApplicationData("bookserverApp", {
+                                sourceData: appData["bookserver-app"],
+                            }),
+                            getRandomAnalysisData(analysisData["analysis_for_openSourceLibraries"])
+                        )
+                    );
+                }
+                analyses.forEach((analysis) => analysis.create());
+            });
+        });
+    });
+
+    it("Perform bulk analysis and validate information on task drawer", function () {
+        // Automates Polarion TC MTA-556
+        Analysis.analyzeAll(analyses[0]);
+
+        cy.get(taskNotificationBadge).click();
+        validateTasks("tech-discovery", [
+            `(tech-discovery) - ${analyses[0].name} - 0`,
+            `(tech-discovery) - ${analyses[1].name} - 0`,
+        ]);
+
+        cy.get("h2.pf-v5-c-notification-drawer__list-item-header-title").contains("analyzer", {
```

Comment:
Could you please refer to a descriptive variable instead of `"h2.pf-v5-c-notification-drawer__list-item-header-title"`?